Shokofin/Providers/Helper.cs

```diff
@@ -4,7 +4,10 @@
 using System.Text.RegularExpressions;
 using Shokofin.API.Models;
 using Title = Shokofin.API.Models.Title;
```

Comment:
I missed this last time but the same comment as I made for line 9. You can remove this.